frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -202,21 +210,23 @@ DarkThemeDefault.args = {
   ...defaultArgs,
   theme: "night",
 };
+DarkThemeDefault.decorators = [DarkBackgroundDecorator];
```

Comment:
Add dark background for all dark theme stories. Actually I only need to add this to dark theme with transparent background, but for simplicity I added to all of them.